### DISCUSSION TITLE
Melhorias na PIPELINE

### DISCUSSION THREAD

**Discussion by Lindemberg-Chagas** [falae-it-talent/falae#17] [General]
![Screenshot_142](https://github.com/falae-it-talent/falae/assets/113239054/49e21cde-7ae5-48f8-bbc1-7c60ebb1ff86)
![Screenshot_143](https://github.com/falae-it-talent/falae/assets/113239054/6df19072-26cd-415c-b2f3-4d7a2df7d22f)
![Screenshot_144](https://github.com/falae-it-talent/falae/assets/113239054/45a5190b-d547-4259-8b2a-738dfaff842b)
![Screenshot_145](https://github.com/falae-it-talent/falae/assets/113239054/37f2a54c-4b95-4564-b191-4178fad0546b)
![Screenshot_146](https://github.com/falae-it-talent/falae/assets/113239054/ee15a12f-f10c-4bde-8a22-90ebb7784cd5)
![Screenshot_147](https://github.com/falae-it-talent/falae/assets/113239054/e1c3c224-8bea-4c56-bb3b-4de8457e66d9)
![Screenshot_148](https://github.com/falae-it-talent/falae/assets/113239054/8963ef50-336c-472f-bc0e-42eb8b4cff35)
![Screenshot_149](https://github.com/falae-it-talent/falae/assets/113239054/df2b6325-410e-4b1d-b1e2-efb7b1015119)
![Screenshot_150](https://github.com/falae-it-talent/falae/assets/113239054/dc454daa-b123-4927-ac05-1be47d8e563d)

**Comment by lucasarags** [comment]
https://github.com/falae-it-talent/falae/actions/runs/7119265715/job/19383979749

Primeira fase concluida, precisa ainda ver questões de mudanças no códigos para secrets e não .env, bem como o restoeDB.

**Reply by Lindemberg-Chagas** [reply]
Boaa Lucas !! trabalhou legal
